dask_jobqueue/core.py

```diff
@@ -294,7 +295,7 @@ def start_workers(self, n=1):
         for _ in range(num_jobs):
             with self.job_file() as fn:
                 out = self._submit_job(fn)
-                job = self._job_id_from_submit_output(out.decode())
+                job = _job_id_from_submit_output(out.decode())
```

Comment:
Are we sure we don't want to leave it as a class method?

Just asking, I don't really have an opinion yet, and I am not a really experienced python user.

If one day we want to be able to add the regex to config file, then won't it be a problem?

Reply:
I also don't have any strong opinion on this.  Just went for the minimal amount of code necessary to implement (and test) this.

Reply:
> - HTCondor (taken somewhere on internet):
> ```
> Submitting job(s)con.
> Logging submit event(s).
> 1 job(s) submitted to cluster 6075.
> 
> # but there is the -terse option that should make it like
> 10744.0 - 10744.2
> ```

Looks like a strong case for leaving the parser a static method of the class.  (This way, it would be easy to create an `HTCondorCluster` with a different parser.

I'll change this.